python/ray/autoscaler/sdk.py

```diff
@@ -216,6 +220,12 @@ def fillout_defaults(config: Dict[str, Any]) -> Dict[str, Any]:
     return fillout_defaults(config)
 
 
+def register_callback_handler(event_name: str,
+                              callback: Callable[[Dict], None]) -> None:
+    """Registers a callback handler for autoscaler events."""
```

Comment:
Can you please add docstring in Google style? (https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html)